templates/next-template/components/ui/dialog.tsx

```diff
@@ -119,6 +119,17 @@ const DialogDescription = React.forwardRef<
 ))
 DialogDescription.displayName = DialogPrimitive.Description.displayName
 
+const DialogClose = React.forwardRef<
```

Comment:
Why not just const DialogClose = DialogPrimitive.Close

Reply:
simplified it!